common/common.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rsa"
+	"fmt"
+	"path/filepath"
+	"reflect"
+	"strings"
+
+	"github.com/lestrrat-go/jwx/jwk"
+	"github.com/spf13/afero"
+	"github.com/veraison/go-cose"
+)
+
+var Fs = afero.NewOsFs()
+
+func SignerFromJWK(rawJWK []byte) (*cose.Signer, error) {
```

Comment:
It would help to add few lines of comment to understand precisely, what this method is doing?

Reply:
ok, done